autoload/ledger.vim

```diff
@@ -674,7 +674,7 @@ endf
 " Use current line as input to ledger entry and replace with output. If there
 " are errors, they are echoed instead.
 function! ledger#entry() abort
-  let l:output = split(system(s:ledger_cmd(g:ledger_main, join(['entry', getline('.')]))), '\n')
+  let l:output = split(system(s:ledger_cmd(g:ledger_main, join(['--', 'entry', getline('.')]))), '\n')
```

Comment:
Shouldn't the `entry` subcommand go on the other side of (prior to) the argument separator?